content/md/cloud/1-set-up-flexibility.md

```diff
@@ -10,47 +10,47 @@ related: access-akeneo-flexibility, get-akeneo-pim-enterprise-archive
 
 The Akeneo cloud instance creation and access are managed on the **projects console** of your [Portal](connect-to-your-portal.html). 
 
-Following the purchase of a flexibility instance, an email is sent to the **setup administrator** inviting them to complete the setup form to create the instance.
+Following the purchase of a flexibility instance, an email is sent to the **setup administrator** inviting them to fill in the setup form to create the instance.
 
-You can set up the instance from the link given in the mail or by accessing the instance configuration page on the **projects console**.
+You can set up the instance from the link you received by email or by accessing the instance configuration page on the **project console**.
 
 ![Flexibility list](../img/serenity_list_waiting_setup.jpg)
 
 ::: info
 The setup should be done for each of your **production** or **sandbox** instances.
 :::
 
-1. First, from the menu, go to `Projects console / Projects` and select your project.
-2. Go to the Cloud instance tab, and open the instance with the status `Waiting for setup`, by clicking on the line of the instance.
+1. First, from the menu, go to `Project console / Projects` and select your project.
+2. Go to the Cloud instance tab, and open the instance with the status `Waiting for setup`, by clicking on the instance line.
 3. Then fill the form with the missing prerequisites in order to create the Cloud server. 
-4. Choose a `Domain prefix`, which is going to be at the beginning of your URL instance (ex: https://yourdomainprefix.cloud.akeneo.com). An instant check is done to see if the name is available.
-5. Select an `Installation locale`, your cloud instance language
-6. Select an `Installation location` for your cloud instance hosting location that is closest to you to reduce latency access.
-7.  Select a `Target number of products` for your catalog.
-8. Once the information is inputted, click `Request an Instance`
+4. Choose a `Domain prefix`, which is going to appear at the beginning of your URL instance (ex: https://yourdomainprefix.cloud.akeneo.com). An instant check runs to see if the name is available.
+5. Select an `Installation locale`, which is your cloud instance language
+6. Select an `Installation location` for your cloud instance hosting location. It should the closest to you to reduce latency access.
```

Comment:
```suggestion
6. Select an `Installation location` for your cloud instance hosting location. It should be the closest to you to reduce latency access.
```

Reply:
Should used for sense of advice
Should be used when you have to need something
Depends on how we want to put it